src/components/sketches/s1s1_neutral.ts

```diff
@@ -0,0 +1,111 @@
+import p5 from 'p5';
+export default function sketch(p: p5): void {
+  let canvas: p5.Renderer;
+
+  const particles: Particle[] = [];
+  const lineLength = 120;
+  const repelRadius = window.innerWidth*0.2;
+
+  p.setup = () => {
+    canvas = p.createCanvas(window.innerWidth, window.innerHeight);
+    canvas.id('p5-background');
+
+    const numParticles = Math.min(Math.floor(window.innerWidth / 10), 200);
+    for (let i = 0; i < numParticles; i++) {
+      particles.push(new Particle());
+    }
+  };
+
+  p.windowResized = () => {
+    p.resizeCanvas(window.innerWidth, window.innerHeight, false);
+  };
+
+  p.draw = () => {
+    p.background(20);
+    p.fill(255, 255, 255, 255);
+
+    p.circle(p.mouseX, p.mouseY, 25);
+
+    particles.forEach((particle, idx) => {
+      particle.update();
+      if (idx % 2 == 0) {
+        particle.draw();
+      } else {
+        particle.draw2();
+      }
```

Comment:
combine the draw functions together and just pass in a boolean

Reply:
nice